ParkingLot/Response.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+
+namespace ParkingLot
+{
+  public class Response
+  {
+    private readonly Car car = null;
+    private readonly List<Car> cars = null;
+    private readonly Ticket ticket = null;
+    private readonly List<Ticket> tickets = null;
+    private readonly OperationStatus operationStatus;
+    private readonly bool isNullTicket;
+    private readonly bool isValidTicket;
+    private string errorMessage = string.Empty;
+
```

Comment:
👍: use response class to store all response content, avoid single response problem